### PR TITLE
include 856u with two blank indicators to fulltext links

### PR DESCRIPTION
**Pull request by banukutlu**
#263

### CODE REVIEW

File: lib/traject/macros/custom.rb [hunk 1]
```diff
@@ -132,7 +132,7 @@ def sought_link_data_exists?(link_type, field)
       end
 
       def fulltext_link_available?(ind1, ind2, url_label)
-        (ind2 == '0' || ind1 == '4') && !NOT_FULLTEXT.match?(url_label)
+        (ind2 == '0' || ind1 == '4' || (ind1.strip.empty? && ind2.strip.empty?)) && !NOT_FULLTEXT.match?(url_label)
```
Comment (cdmo) [inline lib/traject/macros/custom.rb:135]: I'm not sure I see why `strip` is needed. Looking at the spec it looks like an empty string is passed... 🤔

**Review by cdmo — CHANGES_REQUESTED** [review]
One question, other than that a nice tidy PR 👍

File: lib/traject/macros/custom.rb (hunk 1 above)
Comment (banukutlu) [reply lib/traject/macros/custom.rb:135]: yeah this was interesting to me too, the example record I was testing came with whitespace in its indicators like `" "` so I had to add that strip 🤔

Comment (cdmo) [reply lib/traject/macros/custom.rb:135]: ah ok, i was just going by the spec. Did you also see cases where the string was just a true empty string (i.e., `""`)?

Comment (banukutlu) [reply lib/traject/macros/custom.rb:135]: I tried the example records we have in the issue and they all had whitespace in both indicators. of course I am not sure if that is the case always.

```
1174303
23786738
2206901
2187375
```

Comment (cdmo) [reply lib/traject/macros/custom.rb:135]: Ah I was just thrown off by the spec which passes empty string not `" "`

**Comment by cdmo** [comment]
Strange that the build failed in a new way. Did you see that? https://travis-ci.com/github/psu-libraries/psulib_traject/builds/205495909#L794

File: lib/traject/macros/custom.rb (hunk 1 above)
Comment (banukutlu) [reply lib/traject/macros/custom.rb:135]: yeah, I will make the specs like this `" "`

Comment (cdmo) [reply lib/traject/macros/custom.rb:135]: 🙏 danke

**Review by cdmo — CHANGES_REQUESTED** [review]
just needs a rebase

**Review by cdmo — APPROVED** [review]

**Pull request merged**